cw-dex-astroport/tests/astroport_tests.rs

```diff
@@ -266,29 +371,23 @@ mod tests {
     ) -> RunnerResult<()> {
```

Comment:
rename test to `test_stake_and_unstake_native_lp` since it only tests the native case